core/views.py

```diff
@@ -123,15 +124,46 @@ def brother_logout(request):
     logout(request)
     return HttpResponseRedirect('/login')
 
-# TODO: needs to be implemented
-def search(request):
-    query = request.GET.get('query')
-    context = {
-        "result_list": [],
-        'settings': getSettings()
-    }
-    template = loader.get_template('core/search.html')
-    return HttpResponse(template.render(context, request))
+#------------------------------------------------ for search
+class SearchView(ListView):
+    template_name = 'core/search.html'
+    paginate_by = 100
```

Comment:
Where does this variable get used?  I'm thinking that instead of paginating we can just limit the number of results shown and include a "click to view more results" button

Reply:
its a variable used by ListView. I can work around this somehow and try to create pages, but i think the read more results may be a little complicated. I think Im gonna create pages and see how that works first. If it doesnt work out or look right I'll look into a "more results" button.